TheMovieDbTests/TheMovieDbTests.swift

```diff
@@ -6,28 +6,69 @@
 //
 
 import XCTest
+import os
 @testable import TheMovieDb
 
 class TheMovieDbTests: XCTestCase {
+    private static let logger = Logger(subsystem: Constants.subsystemName, category: "TheMovieDbTests")
+    
+    var mockAPIClient: MockAPIClient!
+    var movieModel: MovieAPIModel!
+    var originalNetworkData: [MovieViewModel]!
 
-    override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+    override func setUp() {
+        mockAPIClient = MockAPIClient()
+        let movieApiManager = MovieAPIManager(client: mockAPIClient)
+        movieModel = MovieAPIModel(movieManager: movieApiManager)
+        
+        setOriginalData()
     }
-
-    override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    
+    func setOriginalData() {
+        do {
+            var fileName = "Configuration_example"
+            let configurationExample: ConfigurationWelcome = try FileParser.createMockResponse(filename: fileName)
+            fileName = "Trending_example"
+            let trendingExample: MovieListResult = try FileParser.createMockResponse(filename: fileName)
+            originalNetworkData = trendingExample.results?.map({
+                return MovieViewModel(movie: $0, configuration: configurationExample.image)
+            }) ?? []
+        } catch {
+            Self.logger.error("\(error.localizedDescription)")
+        }
     }
-
-    func testExample() throws {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    
+    func testGetListSuccess() {
+        // Create an expectation
+        let expectation = self.expectation(description: "MovieList")
+        var movieList: [MovieViewModel]?
+        
+        movieModel.getList(movieFeed: MovieFeed.trending) {
```

Comment:
Hi Misa, the tests cant be run because there is an error in this line, caused by some changes that you made in the code that didn't reflect on the tests. Check it out if you have the chance.

Reply:
Hi @ed-pacheco, thanks for your comments. I forgot to re-run the tests after the changes for Search Bar implementation 😔. Will update it

Reply:
Got it, no problem, just make sure that you make the correction before the end of the day